packages/functional-tests/lib/fixtures/standard.ts

```diff
@@ -13,10 +13,21 @@ const DEBUG = !!process.env.DEBUG;
 export { expect };
 export type POMS = ReturnType<typeof createPages>;
 
+type EmailFixtureOptions = {
+  prefix: string; // Prefix for the email address
```

Comment:
Q: What if `prefix` is _optional_, and we default it to an empty string (see my ignorant comments below on L161).
```suggestion
  prefix?: string; // Prefix for the email address
```

Reply:
incorporated. Thanks!